app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java

```diff
@@ -92,22 +93,19 @@ protected void onCreate(Bundle savedInstanceState) {
 
     @OnClick(R.id.buttonSave)
     protected void attemptLogin() {
-        String login = loginView.getText().toString();
+        String login = loginView.getText().toString().trim();
```

Comment:
This can not be trimmed, as discussed in the previous PR, whitespace is a valid character.

Reply:
Ok i will remove that part